src/realm/obj.cpp

```diff
@@ -1936,6 +1936,84 @@ bool Obj::remove_backlink(ColKey col_key, ObjLink old_link, CascadeState& state)
 
 void Obj::assign(const Obj& other)
 {
+    copy(other);
+    auto copy_links = [this, &other](ColKey col) {
+        auto t = m_table->get_opposite_table(col);
+        auto c = m_table->get_opposite_column(col);
+        auto backlinks = other.get_all_backlinks(col);
+        for (auto bl : backlinks) {
+            auto linking_obj = t->get_object(bl);
+            if (c.get_type() == col_type_LinkList) {
```

Comment:
this is missing handling list of primitive mixed values

Reply:
this should now be resolved.

Reply:
This is still missing support for list of primitive mixed links. `c.get_attrs().test(col_attr_List) ...`

Reply:
I must have lost it during the merge.. sorry..